apps/demos/Demos/CardView/CardTemplate/jQuery/index.js

```diff
@@ -1,7 +1,127 @@
+
+    cardEl.append(imageWrapper);
+    cardEl.append(vehicleInfo);
+
+    return cardEl;
+  };
+
+  const createVehicleImg = (vehicle) => {
+    const imageWrapper = $('<div>').addClass('vehicle__img-wrapper');
+    const img = $('<img>').addClass('vehicle__img');
+    img.attr({
+      src: `../../../../images/vehicles/image_${vehicle.ID}.png`,
+      alt: `${vehicle.TrademarkName} ${vehicle.Name}`,
+    });
+
+    imageWrapper.append(img);
+
+    return imageWrapper;
+  };
+
+  const createVehicleInfo = (card) => {
+    const vehicleInfo = $('<div>').addClass('vehicle__info');
+
+    const name = `${card.data.TrademarkName} ${card.data.Name}`;
+    const vehicleName = $('<div>').addClass('vehicle__name').text(name).attr('title', name);
+
+    const priceInfo = $('<div>').addClass('vehicle__price').text(`$${card.data.Price}`);
+
+    const typeContainer = $('<div>').addClass('vehicle__type-container');
+    const typeInfo = $('<div>').addClass('vehicle__type').text(`${card.data.CategoryName}`);
+    typeContainer.append(typeInfo);
+
+    const specContainer = $('<div>').addClass('vehicle__spec-container');
+    const modificationInfo = $('<div>').addClass('vehicle__modification').text(`${card.data.Modification}`);
+    const bodyInfo = $('<div>').addClass('vehicle__modification')
+      .text(`${card.data.BodyStyleName} ${card.data.Horsepower} h.p.`);
+    specContainer.append(modificationInfo);
+    specContainer.append(bodyInfo);
+
+    const popupContentTemplate = function () {
+      const sourceLink = `http://${card.data.Source}`;
+      return $('<div>').append(
+        $(`<p><b>Image licensed under:</b> <span>${card.data.LicenseName}</span></p>`),
+        $(`<p><b>Author:</b> <span>${card.data.Author}</span></p>`),
+        $(`<p><b>Source link:</b> <a href='${sourceLink}' class='license__link'>${sourceLink}</a></p>`),
+        $(`<p><b>Edits:</b> <span>${card.data.Edits}</span></p>`),
+      );
+    };
+
+    const footerContainer = $('<div>').addClass('vehicle__footer-container');
+    const footerButton = $('<div>').dxButton({
+      text: 'Image Info',
+      type: 'default',
+      width: '100%',
+      onClick() {
+        popup.option({
+          contentTemplate: () => popupContentTemplate(),
+        });
+        popup.show();
+      },
+    });
+    footerContainer.append(footerButton);
+
+    vehicleInfo.append(vehicleName);
+    vehicleInfo.append(priceInfo);
+    vehicleInfo.append(typeContainer);
+    vehicleInfo.append(specContainer);
+    vehicleInfo.append(footerContainer);
+
+    return vehicleInfo;
+  };
+
+  $(function () {
+    $("#card-view").dxCardView({
+      dataSource: vehicles,
+      cardsPerRow: 'auto',
+      cardMinWidth: 260,
+      cardMaxWidth: 260,
+      paging: {
+        pageSize: 12,
+      },
+      columns: [
+        "TrademarkName",
+        "Name",
+        {
+          dataField: "Price",
+          format: 'currency',
+        },
+        "CategoryName",
+
+        "Modification",
```

Comment:
Let's remove this blank line
```suggestion
        "CategoryName",
        "Modification",
```